modernizr/modernizr.d.ts

```diff
@@ -96,9 +97,8 @@ interface ModernizrStatic {
     load(resourceObject: any): void;
     load(resourceString: string): void;
 
-    prefixed(): boolean;
-    prefixed(property: string): boolean;
-    prefixed(property: string, obj: any, element?: any): boolean;
+    prefixed(property: string): any;
+    prefixed(property: string, obj: any, element?: any): any;

```

Comment:
Yes, it may return any JavaScript Object as stated here: http://modernizr.com/docs/#prefixed... so the type really cannot be determined.